semcore/data-table/src/DataTable.tsx

```diff
@@ -258,7 +261,13 @@ class RootDefinitionTable extends Component<AsProps> {
           children,
           ref: column?.props?.ref || React.createRef(),
         },
-      });
+      } as Column;
+
+      // @ts-ignore
```

Comment:
Why do u use ts-ignore? 🤨

Reply:
<img width="653" alt="Screenshot 2022-10-19 at 11 26 34" src="https://user-images.githubusercontent.com/4582799/196637979-8c91db01-c386-42c4-bd2a-cf96580c6ce0.png">
Without it, TS go crazy 🤯

Reply:
Ts is not going crazy. You just provided wrong typings for `column` variable.

You have either initialize it with corresponding value (`let columns: Cloumn[] = []`) or provide corresponding typings (`let columns: Column[] | undefined`).

Reply:
It became clearer. The error was not informative enough for me(